opentelemetry-sdk/src/opentelemetry/sdk/trace/export/__init__.py

```diff
@@ -220,6 +226,24 @@ def on_end(self, span: ReadableSpan) -> None:
             with self.condition:
                 self.condition.notify()
 
+    def _at_fork_reinit(self):
+        # could be in an inconsistent state after fork, reinitialise by calling `_at_fork_reinit`
+        # (creates a new lock internally https://github.com/python/cpython/blob/main/Python/thread_pthread.h#L727)
+        # if exists, otherwise create a new one.
+        if hasattr(self.condition, "_at_fork_reinit"):
+            self.condition._at_fork_reinit()
+        else:
+            self.condition = threading.Condition(threading.Lock())
+
+        self.queue.clear()
```

Comment:
Curious, I understand that we want to clear the queue when the process is forked, but will this clear the current queue if there are entries in it, or is the hook called AFTER the process is forked?